main.go

```diff
@@ -259,13 +260,41 @@ func migrate(db *gorm.DB) {
 	}
 }
 
-func initializeLogger(config *configuration.Data) (func(), error) {
+func initializeLogger(config *configuration.Data, userService user.Service) (func(), error) {
 	// Initialized developer mode flag and log level for the logger
 	log.InitializeLogger(config.IsLogJSON(), config.GetLogLevel())
+	sentryDSN := config.GetSentryDSN()
 
 	// Initialize sentry client
 	return sentry.InitializeSentryClient(
+		&sentryDSN,
 		sentry.WithRelease(controller.Commit),
 		sentry.WithEnvironment(config.GetEnvironment()),
-	)
+		sentry.WithUser(extractUserInfo(userService)))
+}
+
+func extractUserInfo(userService user.Service) func(ctx context.Context) (*raven.User, error) {
+	return func(ctx context.Context) (*raven.User, error) {
+		userToken := goajwt.ContextJWT(ctx)
+		if userToken == nil {
+			return nil, fmt.Errorf("no token found in context")
+		}
+		ttoken := &controller.TenantToken{Token: userToken}
+		user, err := userService.GetUser(ctx, ttoken.Subject())
```

Comment:
You don't have to call Auth service to get User Info. You can get it from the token.
userID := ttoken.Subject()
username := ttoken.Username()
email := ttoken.Email()

Reply:
I didn't realize this - I sometimes get lost in these auth/token things. thanks for pointing it out :+1: 
fixed in 4d77efe